lib/teleterm/daemon/config.go

```diff
@@ -25,10 +27,28 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type (
+	cluster interface {
+		GetServers(ctx context.Context) ([]clusters.Server, error)
+	}
+	storage interface {
+		GetByURI(uri string) (cluster, error)
+	}
+)
```

Comment:
nit: de-group interface declaration: less indent, a bit more common as an idiom.

```suggestion
type cluster interface {
	GetServers(ctx context.Context) ([]clusters.Server, error)
}

type storage interface {
	GetByURI(uri string) (cluster, error)
}
```

Reply:
Yah, sorry, my editor auto formatted here ahhaa.